src/pages/Index.jsx

```diff
@@ -0,0 +1,16 @@
+// import React from 'react'
+import Categories from "../components/Categories/Categories";
+import CompanyShowcaseComponent from "../components/companyShowcaseComponent/CompanyShowcaseComponent";
+import Hero from "../components/Hero/Hero";
+import JobsShowcase from "../components/JobsShowcase/JobsShowcase";
+import LatestJobs from "../components/LatestJob/LatestJobs";
+import { showCaseData } from "../data/showCaseData";
+
+export {
+  Hero,
+  JobsShowcase,
+  Categories,
+  LatestJobs,
+  CompanyShowcaseComponent,
+  showCaseData,
+};
```

Comment:
Great work on importing the components for the home page but you need to return them as jsx and export default the whole page.
For example:
```
const Home = () => {
return(
<div>
<Hero />
<JobsShowcase />
...
..
</div>
)};
export default Home;
```